src/ext.rs

```diff
@@ -115,8 +117,54 @@ impl HeaderCaseMap {
 
     pub(crate) fn append<N>(&mut self, name: N, orig: Bytes)
     where
-        N: IntoHeaderName,
+        N: IntoHeaderName + std::fmt::Display,
```

Comment:
This can be removed now, right?

Reply:
Yep, I had it in there for some printf debuging

Reply:
Done